packages/ui/list/paginator.tsx

```diff
@@ -10,6 +10,10 @@ export function Paginator() {
   const end = endMax < totalCount ? endMax : totalCount;
   const t = useTranslations('Paginator');
 
+  if (!totalCount || totalCount === 0) {
```

Comment:
If `totalCount` = 0, it succeeds on the first check (`!totalCount`) and not get to the second - right? If the second is import, rephrase the first check (e.g. `(totalCount === undefined)`)?